### PR TITLE
bash: add logoutExtra option

### PR DESCRIPTION
**Pull request by Gerschtli**

### CODE REVIEW

File: modules/programs/bash.nix [hunk 1]
```diff
@@ -195,6 +204,12 @@ in
         ${cfg.bashrcExtra}
       '';
 
+      home.file.".bash_logout".text = ''
+        # -*- mode: sh -*-
```
Comment (teto) [inline modules/programs/bash.nix]: is that an emacs modeline ? I don't see the point  of adding it here. 
I would rather see no file created if no extra is specified

File: modules/programs/bash.nix [hunk 2]
```diff
@@ -123,6 +123,15 @@ in
           interactive shell.
         '';
       };
+
+      logoutExtra = mkOption {
```
Comment (teto) [inline modules/programs/bash.nix]: I would prefer `logoutExtraConfig` to remain consistant.

Comment (Gerschtli) [reply modules/programs/bash.nix]: I named it `logoutExtra` to remain consistant with `initExtra`, `bashExtra` etc

File: modules/programs/bash.nix (hunk 1 above)
Comment (Gerschtli) [reply modules/programs/bash.nix]: Fixed it

**Comment by Gerschtli** [comment]
@teto is this PR okay? You didn't resolve your requested changes.

**Pull request merged**